src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -31,12 +29,13 @@ public static String getAddCommand(Person person) {
     public static String getPersonDetails(Person person) {
         StringBuilder sb = new StringBuilder();
         sb.append(PREFIX_NAME + person.getName().fullName + " ");
-        sb.append(PREFIX_PHONE + person.getPhone().value + " ");
+        sb.append(PREFIX_CLASS_GROUP + person.getClassGroup().classGroup + " ");
         sb.append(PREFIX_EMAIL + person.getEmail().value + " ");
-        sb.append(PREFIX_ADDRESS + person.getAddress().value + " ");
-        person.getTags().stream().forEach(
-            s -> sb.append(PREFIX_TAG + s.tagName + " ")
-        );
+        sb.append(PREFIX_PHONE + person.getPhone().value + " ");
+        sb.append(PREFIX_TELEGRAM + (person.getTelegram().isPresent()
+                ? person.getTelegram().get().telegramId : "") + " ");
```

Comment:
Referring to my suggestion in Telegram.java,
`sb.append(PREFIX_TELEGRAM + person.getTelegram().orElse(Telegram::EMPTY)`
Can we rely on the toString override to append to the string builder instead?